rpc/block_test.go

```diff
@@ -163,11 +173,11 @@ func TestBlockWithTxHashes(t *testing.T) {
 		if err != test.ExpectedError {
 			t.Fatal("BlockWithTxHashes match the expected error:", err)
 		}
-		switch resultBlock := result.(type) {
-		case Block:
-			block, ok := result.(*Block)
+		switch result.(type) {
```

Comment:
Also, we should add a default. This method returns an interface, which could be anything. We should check for this